e2e/api.spec.ts

```diff
@@ -4,14 +4,11 @@ import { electronTest } from './helpers'
 
 declare const window: { api: typeof import('../src/backend/api').default }
 
-electronTest('renders the first page', async (app) => {
-  const page = await app.firstWindow()
+electronTest('renders the first page', async (app, page) => {
```

Comment:
I'm passing `app` and `page` from the helper to avoid calling `const page ....` for every test.